website/views/shared/tasks/task_view/index.jade

```diff
@@ -37,7 +37,8 @@
     label(for='box-{{::obj._id}}_{{::task._id}}')
 
 // main content
-.task-text(ng-dblclick='task._editing ? saveTask(task) : editTask(task)')
+// don't allow a double click to edit if the object is locked
+.task-text(ng-dblclick='obj._locked ? {} : task._editing ? saveTask(task) : editTask(task, obj)')
```

Comment:
This is much cleaner, thanks. I don't know why it didn't cross my mind to make a new method as a pass through to only effect this call. Apparently I'm off my game today